R/RLearner_fcregr_autoarima.R

```diff
@@ -0,0 +1,95 @@
+makeRLearner.fcregr.auto.arima = function() {
+  makeRLearnerForecastRegr(
+    cl = "fcregr.auto.arima",
+    package = "forecast",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "d", lower = 0, upper = Inf),
+      makeIntegerLearnerParam(id = "D", lower = 0, upper = Inf),
+      makeIntegerLearnerParam(id = "max.p", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.q", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.P", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.Q", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.order", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.d", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.D", lower = 0, upper = Inf, default = 1),
+      makeIntegerLearnerParam(id = "start.p", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "start.q", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "start.P", lower = 0, upper = Inf, default = 1),
+      makeIntegerLearnerParam(id = "start.Q", lower = 0, upper = Inf, default = 1),
```

Comment:
For clarification, I want the user to set the upper bound, but they should be able to tune over start.P, etc. If I set that parameter to `tunable = FALSE` will they still be able to tune it?